src/patternfly/components/AppLauncher/docs/code.md

```diff
@@ -0,0 +1,30 @@
+## Accessibility
+
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `aria-expanded="false"` | `.pf-c-app-launcher__toggle` |  Indicates that the menu is hidden |
+| `aria-expanded="true"` | `.pf-c-app-launcher__toggle` |  Indicates that the menu is visible |
+| `aria-label="Actions"` | `.pf-c-app-launcher__toggle` | Provides an accessible name for the app launcher when an icon is used. **Required** |
+| `aria-hidden="true"` | `.pf-c-app-launcher__toggle-icon` | Hides the icon from assistive technologies |
+| `hidden` | `.pf-c-app-launcher__menu` | Indicates that the menu is hidden so that it isn't visible in the UI and isn't accessed by assistive technologies |
+| `aria-expanded="true"` | `.pf-c-app-launcher__menu` | Indicates that the menu is visible |
+| `disabled` | `button.pf-c-app-launcher__menu-item` | When the menu item uses a button element, indicates that it is unavailable and removes it from keyboard focus |
+| `aria-disabled="true"` | `a.pf-c-app-launcher__menu-item` | When the menu item uses a link element, indicates that it is unavailable |
+| `tabindex="-1"` | `a.pf-c-app-launcher__menu-item` | When the menu item uses a link element, removes it from keyboard focus |
+
+## Usage
+
+| Class | Applied | Outcome |
+| -- | -- | -- |
+| `.pf-c-app-launcher` | `<div>` | Defines the parent wrapper of the app launcher. |
+| `.pf-c-app-launcher__toggle` | `<button>` | Defines the app launcher toggle |
+| `.pf-c-app-launcher__toggle-icon` | `<i>` | Defines the app launcher toggle icon |
+| `.pf-c-app-launcher__toggle-text` | `<span>` | Defines the app launcher toggle text |
+| `.pf-c-app-launcher__menu` | `<div>` | Defines the parent wrapper of the menu items |
+| `.pf-c-app-launcher__menu-item` | `<a>` | Defines a menu item that navigates to another page |
+| `.pf-c-app-launcher__menu-item` | `<button>` | Defines a menu item that performs an action on the current page |
+| `.pf-m-expanded` | `.pf-c-app-launcher` | Modifies for the expanded state |
+| `.pf-m-hover` | `.pf-c-app-launcher__menu-item`, `.pf-c-app-launcher__toggle` | Forces display of the hover state of the element. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:hover` pseudo-class. |
+| `.pf-m-focus` | `.pf-c-app-launcher__menu-item` | Forces display of the focus state of the element. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:focus` pseudo-class. |
+| `.pf-m-active` | `.pf-c-app-launcher__toggle` | Forces display of the active state of the element. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:active` pseudo-class. |
```

Comment:
currently these styles are on `.pf-c-app-launcher__toggle-icon` but I think they should be moved to `.pf-c-app-launcher__toggle`